slider/tests/test_beatmap.py

```diff
@@ -142,6 +142,8 @@ def test_parse_section_hit_objects(beatmap):
 def test_hit_objects_stacking():
```

Comment:
Might be worth asserting these in the `test_parse_section_hit_objects` function above too -
```py
assert hit_objects_0.new_combo
assert not hit_objects_0.combo_skip
```

Reply:
True, added